src/main/java/cz/lundegaard/form/controller/PersonController.java

```diff
@@ -80,7 +81,7 @@ public Person updatePerson(@PathVariable long id, @RequestBody Person person) th
     public ResponseEntity deletePerson(@PathVariable long id) {
         try {
             personService.deletePerson(id);
-        } catch (Exception e) {
+        } catch (ResourceNotFoundException e) {
             return ResponseEntity.notFound().build();
```

Comment:
Dokonce myslim tady neni treba vyjimky chytat a meni response, annotation @ResponseStatus(value = HttpStatus.NOT_FOUND) na vyjimce by to mela resit za vas